plugins/techdocs-backend/src/service/router.ts

```diff
@@ -58,14 +59,20 @@ export async function createRouter({
     const { '0': path } = req.params;
     const entityName = getEntityNameFromUrlPath(path);
 
-    publisher
-      .fetchTechDocsMetadata(entityName)
-      .then(techdocsMetadataJson => {
-        res.send(techdocsMetadataJson);
-      })
-      .catch(reason => {
-        res.status(500).send(`Unable to get Metadata. Reason: ${reason}`);
-      });
+    try {
+      const techdocsMetadata: TechDocsMetadata = await publisher.fetchTechDocsMetadata(
+        entityName,
+      );
```

Comment:
Since `publisher.fetchTechDocsMetadata` returns a typed object, by just doing `const techdocsMetadata = await publisher.fetchTechDocsMetadata(entityName);`, `techdocsMetadata` should get the correct type.

Reply:
That makes a lot of sense! I wasn't aware whether an explicit type annotation here would make it more readable or add a bit more context to what is going on; I do think it adds a bit of noise though :) 

Done!